doc/book/connectors/__python.rst

```diff
@@ -44,3 +44,214 @@ Also there are several community-driven Python connectors:
+
+    *   -   Documentation
+        -   Yes (`github.io <https://igorcoding.github.io/asynctnt/>`__)
+        -   No
+        -   No
+        -   Yes (`tarantool.io
+            <https://www.tarantool.io/en/doc/latest/getting_started/getting_started_python/>`__
+            and `readthedocs
+            <https://tarantool-python.readthedocs.io/en/latest/quick-start.en.html>`__
+            (`obsolete
+            <https://github.com/tarantool/tarantool-python/issues/67>`__))
+
+    *   -   Testing / CI / CD
+        -   Github Actions
+        -   No (tests exists)
+        -   No (tests exists)
+        -   Travis CI and Appveyor
+
+    *   -   Github Stars
+        -   51
+        -   38
+        -   17
+        -   78
+
+    *   -   Static Analysis
+        -   Yes (Flake8)
+        -   No
+        -   No
+        -   No
+
+    *   -   Packaging
+        -   `Pip <https://pypi.org/project/asynctnt/>`__
+        -   `Pip <https://pypi.org/project/aiotarantool/>`__
+        -   `Pip <https://pypi.org/project/gtarantool/>`__
+        -   `Deb, RPM, Pip <https://github.com/tarantool/tarantool-python#download-and-install>`__
+
+    *   -   Code coverage
+        -   Yes (no visualization)
+        -   No
+        -   No
+        -   Yes (no visualization)
+
+    *   -   Support asynchronous mode
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes? (`gevent
+            <https://www.gevent.org/api/gevent.event.html#gevent.event.AsyncResult>`__,
+            example: `test_gevent.py
+            <https://github.com/shveenkov/gtarantool/blob/master/tests/test_gevent.py>`__)
+        -   No
+
+    *   -   Batching support
+        -   No
+        -   No
+        -   No
+        -   No (`issue #55 <https://github.com/tarantool/tarantool-python/issues/55>`__)
+
+    *   -   Schema reload, space and index names
+        -   Yes (auto_refetch_schema)
+        -   Yes (handle *SchemaReloadException*)
+        -   No
+        -   Yes (handle *SchemaReloadException*)
+
+    *   -   Schema reload, Access tuple fields by names
+        -   Yes
+        -   FIXME
+        -   FIXME
+        -   FIXME
+
+    *   -   `SQL support <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`__
+        -   Yes (tests/test_op_sql.py)
+        -   No (using call server-side functions)
+        -   No
+        -   Yes (tarantool/connection.py)
+
+    *   -   `Interactive transactions <https://www.tarantool.io/en/doc/latest/book/box/stream/>`__
+        -   No
+        -   No
+        -   No
+        -   No (`issue #163 <https://github.com/tarantool/tarantool-python/issues/163>`__)
+
+    *   -   `Varbinary support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   No
+        -   No
+        -   No
+        -   No
```

Comment:
AFAIR, tarantool-python has some support of varbinary (but there are questions how it is mapped into python types). I would also re-check other connectors here.

(It is more a note for myself.)

Reply:
We can say 'questionable' and link https://github.com/tarantool/tarantool-python/issues/105.

Reply:
Discussed and decided to recheck varbinary support in tarantool-python.

Reply:
Replaced with 'Yes'.